### PR TITLE
cmd/scollector: Add fortinet collector

### PR DESCRIPTION
**Pull request by kylebrandt**

### CODE REVIEW

File: cmd/scollector/collectors/snmp_fortinet.go [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+	"fmt"
+	"strconv"
+	"time"
+
+	"bosun.org/cmd/scollector/conf"
+	"bosun.org/metadata"
+	"bosun.org/opentsdb"
+)
+
+const (
```
Comment (captncraig) [inline cmd/scollector/collectors/snmp_fortinet.go]: can we define these inside the c_fortinet_os function plz?

**Comment by captncraig** [comment]
lgtm otherwise

File: cmd/scollector/collectors/snmp_fortinet.go (hunk 1 above)
Comment (kylebrandt) [inline cmd/scollector/collectors/snmp_fortinet.go]: @captncraig  That would be change form all the other snmp collectors. We can do that, but would rather it be a commit changing all of them.

**Comment by kylebrandt** [comment]
pushed manually to master

**Pull request closed without merging**